values/keycloak/keycloak-otomi-db.gotmpl

```diff
@@ -1,11 +1,33 @@
 {{- $v := .Values }}
 {{- $k := $v.apps.keycloak }}
 {{- $cnpg := $v.apps.cnpg }}
+{{- $sp := $cnpg.storage }}
 {{- $kdb := $v.databases.keycloak }}
+{{- $obj := $v.obj.provider }}
+{{- $bu := $v.obj.bucket }}
```

Comment:
This should prevent installation errors when obj.provider is set to "disabled"

```suggestion
{{- if ne $obj.type "disabled" }}
  {{- $bu := $v.obj.bucket }}
{{- end }}
```

Reply:
Solved in [c3e1a9d](https://github.com/linode/apl-core/pull/1640/commits/c3e1a9d623f105fe0f37588d124edebb808cf737)